content/_data/changelogs/lts.yml

```diff
@@ -342,6 +342,156 @@
       pull: 2868
       issue: 44010
 
+- version: "2.60.1"
+  date: 2017-06-21
+  lts_predecessor: "2.46.3"
+  lts_baseline: "2.60"
+  banner: >
+    <strong>2.60.1 is the first Jenkins LTS release that requires Java 8 to run.</strong>
+    If you're using the Maven Project type, please note that it needs to use a JDK capable of running Jenkins, i.e. JDK 8 or up.
+    If you configure an older JDK in a Maven Project, Jenkins will attempt to find a newer JDK and use that automatically.
+    If your SSH Slaves fail to start and you have the plugin install the JRE to run them, make sure to update SSH Slaves Plugin to at least version 1.17.
+  lts_changes:
+    - type: major rfe
+      message: >
+        <strong>Jenkins (master and agents) now requires Java 8 to run.</strong>
+      references:
+        - issue: 27624
+        - issue: 42709
+        - pull: 2802
+        - url: /blog/2017/04/10/jenkins-has-upgraded-to-java-8/
+          title: announcement blog post
+    - type: bug
+      message: >
+        Update Groovy to 2.4.8 to address memory leak issue.
+        <strong><em>Pipeline: Groovy</em> needs to be upgraded to 2.28 or higher to prevent regressions.</strong>
+      references:
+        - issue: 33358
+        - issue: 42189
+
+
+    # WINDOWS SERVICES START
+    - type: major rfe
+      message: >
+        Upgrade the Windows Agent Installer module from 1.6 to 1.7.
+        This change picks major updates in Windows service management logic.
+      references:
+        - url: https://github.com/jenkinsci/windows-slave-installer-module/blob/master/CHANGELOG.md#17
+          title: full changelog
+        - url: https://github.com/jenkinsci/windows-slave-installer-module#upgrading-old-agents
```

Comment:
We still have 10 days for this, and as contributor to this repo you have write access to the PR branch in my fork.